.github/workflows/generate-command-reference.yml

```diff
@@ -0,0 +1,29 @@
+name: Automatically generate command reference topics
+on:
+  push:
+  workflow_dispatch:
+
+env:
+  AMC_COMMAND_REFERENCE_DIR: "./reference/cmd-ref/amc-command-reference"
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-24.04
+    steps:
+    - name: Setup Anbox Cloud
+      uses: canonical/anbox-cloud-github-action@main
+      with:
```

Comment:
We should set a channel based on what is in `.base_version`. See https://github.com/canonical/anbox-cloud-docs/blob/main/.github/workflows/update-api-specs.yaml#L22